src/main/java/browserview/BrowserComponent.java

```diff
@@ -47,11 +46,17 @@
     private static final String CHROME_DRIVER_LOCATION = "browserview/";
     private static final String CHROME_DRIVER_BINARY_NAME = determineChromeDriverBinaryName();
 
-    private String pageContentOnLoad = "";
-
     private static final int SWP_NOSIZE = 0x0001;
     private static final int SWP_NOMOVE = 0x0002;
     private static final int SWP_NOACTIVATE = 0x0010;
+
+    /**
+     * The two variables below are used to wait for a webpage to load, where a wait consist of
+     * Thead.sleep() by sleep time, repeated until the webpage is loaded or max retry is reached.
+     */
```

Comment:
I don't there there is any need to mention the implementation details. One variable specifies the duration between tries and the other specifies how many tries.